tests/extras/test_full_text_ocr.py

```diff
@@ -0,0 +1,48 @@
+import json
+
+import pytest
+
+from mindee import AsyncPredictResponse
+from mindee.product import InternationalIdV2
+from tests.utils import EXTRAS_DIR
+
+# NOTE: Implementing extras per pages without content (like the Java library)
+# would be a breaking change for the Python SDK.
+# This fixture is left here as a reminder that next major version should probably implement it.
+
+# @pytest.fixture
+# def load_pages():
+#     def _load_pages():
+#         with open(EXTRAS_DIR / "full_text_ocr/complete.json", "r") as file:
+#             prediction_data = json.load(file)
+#         print("PData", AsyncPredictResponse(InternationalIdV2, prediction_data).document.inference.pages)
+#         return AsyncPredictResponse(InternationalIdV2, prediction_data).document.inference.pages
+#
+#     return _load_pages
+
+
+@pytest.fixture
+def load_document():
+    def _load_document():
+        with open(EXTRAS_DIR / "full_text_ocr/complete.json", "r") as file:
+            prediction_data = json.load(file)
+
+        return AsyncPredictResponse(InternationalIdV2, prediction_data).document
+
+    return _load_document
```

Comment:
You should return a value rather than a callable.

If you need to pass a variable to your fixture, it can be done as follows:
https://docs.pytest.org/en/stable/example/parametrize.html#indirect-parametrization

Reply:
Yeah Idk I wanted to try something new :laughing: I'll put it back like it used to be